scripts/scripts/testing/automations/batch/india.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pandas as pd
+from cowidev.utils.web import request_json
+from cowidev.utils.utils import get_project_dir
+
+
+class India:
+    location: str = "India"
+    units: str = "samples tested"
+    source_label: str = "Indian Council of Medical Research"
+    source_url: str = "https://raw.githubusercontent.com/datameet/covid19/master/data/icmr_testing_status.json"
+    source_url_ref: str = "https://github.com/datameet/covid19"
+    notes: str = "Made available by DataMeet on GitHub"
+    rename_columns: dict = {"report_time": "Date", "samples": "Cumulative total"}
+
+    def read(self) -> pd.DataFrame:
+        data = request_json(self.source_url)
+        df = pd.DataFrame.from_records(data["rows"])["value"].apply(pd.Series)
+        df = df.drop(columns=["_id", "_rev", "individuals", "confirmed_positive", "source", "type"])
+        df["report_time"] = df["report_time"].str.slice(0, 10)
```

Comment:
My proposal here is

```python
data = [x['value'] for x in data['rows']]  # Tweak data
df = pd.DataFrame.from_records(data, columns=['report_time', 'samples'])  # Load only columns needed
df['report_time'] = clean_date_series(df['report_time'], '%Y-%m-%dT%H:%M:%S.%f%z')  # Clean date
```

Will need to import `cowidev` function:

```
from cowidev.utils.clean import clean_date_series
```

Cleaning the date this way is safer and more robust to possible source format changes.

Reply:
Thanks